zeppelin-distribution/src/assemble/distribution.xml

```diff
@@ -97,6 +97,37 @@
     <fileSet>
       <outputDirectory>/lib/interpreter</outputDirectory>
       <directory>../zeppelin-interpreter/target/lib</directory>
+      <excludes>
```

Comment:
I think we can just remove this line. https://github.com/apache/zeppelin/blob/master/bin/interpreter.sh#L88

shaded jar (zeppelin-interpreter-api) is only used for interpreter process. Zeppelin server process still use the unshaded jar (zeppelin-interpreter)

Reply:
1. The "${ZEPPELIN_HOME}/zeppelin-interpreter-api/target" path only exists when running travis CI, so bin/interpreter.sh allows the CI to pass the test.
2. There is no "${ZEPPELIN_HOME}/zeppelin-interpreter-api/target" path in zeppelin-0.X.tar.gz released by zeppelin-distribution.
3. If you do not modify zeppelin-distribution/src/assemble/distribution.xml, there will be no zeppelin-interpreter-api-XXX.jar in zeppelin-0.X.tar.gz

Reply:
OK.